log/log.go

```diff
@@ -0,0 +1,63 @@
+package log
```

Comment:
note: for slightly more complex things subpackages of runtime also makes sense...

I'm just trying to avoid blowing up the root folder with small things... Honestly, I'm tempted to move atomics too... either into runtime, or making some utilities package... for all the small stuff..